x11/rxvt-unicode/Portfile

```diff
@@ -18,44 +18,41 @@ long_description \
                 completely flicker-free \
                 display, support for Xft and core fonts in any \
                 combination, etc.
-platforms       darwin
 homepage        http://software.schmorp.de/pkg/rxvt-unicode.html
 master_sites    http://dist.schmorp.de/rxvt-unicode/ \
                 http://dist.schmorp.de/rxvt-unicode/Attic/
 use_bzip2       yes
 
-checksums       rmd160  ce20bcda75cadd4a22edf166ce50384080de1b23 \
-                sha256  643116b9a25d29ad29f4890131796d42e6d2d21312282a613ef66c80c5b8c98b \
-                size    904950
+checksums       rmd160  a2201a7ff63283715fe7230e17291b5c12386c1b \
+                sha256  fe1c93d12f385876457a989fc3ae05c0915d2692efc59289d0f70fabe5b44d2d \
+                size    877186
 depends_build \
         port:pkgconfig
 
 # urxvt also depends on xorg-libX11, xrender and fontconfig but all these ports are in the dependency tree of Xft2
 depends_lib \
+        port:libptytty \
         port:Xft2 \
         port:startup-notification \
         port:xorg-libXmu \
         port:ncurses \
-        port:perl5.30
+        port:perl5.34
```

Comment:
Please see #15265 for the "correct" changes.

Reply:
Fixed, thanks.